### DISCUSSION TITLE
BBC Three

### DISCUSSION THREAD

**Discussion by Ianszh** [Ascoware/get-iplayer-automator#379] [General]
Hello,

Great app, will you be adding BBC Three anytime soon?

Thanks

**Comment by skovatch** [comment]
You can get BBC Three shows by going to the show's page and choosing "Use Current Webpage" from Get iPlayer Automator. BBC Three shows aren't indexed by get_iplayer, so you can't search for them. I'm not clear as to why that's the case, but that's the reason you won't be able to search for shows.

**Reply by Ianszh** [reply]
Ok thanks, I’ll do it like that.

**Comment by skovatch** [comment]
BBC Three is now a 'supported' channel in get_iplayer, so the option to search just BBC Three is back in the preferences.
